templates/dashboard.html.twig

```diff
@@ -0,0 +1,118 @@
+{% extends '@EasyAdmin/default/layout.html.twig' %}
+
+{% block page_title %}
+  {{ ('dashboard.'~entityType)|trans }}
+{% endblock %}
+
+{% block content_title %}
+  {{ ('dashboard.'~entityType)|trans }}
+{% endblock %}
+
+{% block global_actions_wrapper %}
+  <div class="global-actions w-75 dashboard-filters" style="justify-content: flex-end">
+    {{ form_start(filters, { attr: { class: 'w-100' } }) }}
+    <div class="row w-100 flex-row">
+      <div class="col">
+        {{ form_widget(filters.groups) }}
+      </div>
+      <div class="col">
+        {{ form_widget(filters.subowner) }}
+      </div>
+      <div class="col">
+        {{ form_widget(filters.theme) }}
+      </div>
+      <div class="col">
+        {{ form_widget(filters.category) }}
+      </div>
+      <div class="col-4">
+        {{ form_widget(filters.search) }}
+      </div>
+    </div>
+    {{ form_end(filters) }}
+  </div>
+{% endblock %}
+
+{% block main %}
+  <div class="dashboard--table">
+    <table class="table table-bordered table-striped table-sm">
+      <thead class="">
+      <th></th>
+      {% for item in paginator %}
+        <th class="rotate">
+          <div>
+          <span>
+            <a href="{{ path('easyadmin', { action: 'show', id: item.id, entity: entityType|capitalize }) }}" title="{{ item.showableName }}">
+              {{ breakintolines(item.showableName, 35, 2) | raw }}
+            </a>
+          </span>
+          </div>
+        </th>
+      {% endfor %}
+      <th class="table-right-column"></th>
+      </thead>
+      <tbody>
+      {% if categories|length > 0 %}
+        {% for category in categories %}
+          <tr>
+            <td><h3><strong>{{ category.name }}</strong></h3></td>
+            {% for result in paginator %}
+              <td></td>
+            {% endfor %}
+            <td class="table-right-column"></td>
+          </tr>
+          {% for question in (category.questions)|sort_order %}
+            <tr>
+              <td>{{ question.question }}</td>
+              {% for result in paginator %}
+                <td class="dashboard--table-answer">
+                  <div class="dashboard--table-cell">
+                    {% set answer = getanswer(result, question) %}
+                    {% if answer is not null %}
+                      {% include 'easy_admin_overrides/field_smiley.html.twig' with { 'value': answer.smiley, 'title': answer.note, 'width': 30 } %}
+                    {% endif %}
+                  </div>
+                </td>
+              {% endfor %}
+              <td class="table-right-column"></td>
+            </tr>
+          {% endfor %}
+          {% if not loop.last %}
+            <tr>
+              <td class="dashboard--table-spacing" colspan="{{ (paginator|length) + 2 }}">&nbsp;</td>
+            </tr>
+          {% endif %}
+        {% endfor %}
+      {% else %}
+        <tr>
+          <td colspan="100%">
```

Comment:
Maybe I'm getting old, but shouldn't the value of `colspan` be a number (of cells to span)?